backend/pkg/console/endpoint_compatibility.go

```diff
@@ -125,6 +125,12 @@ func (s *Service) GetEndpointCompatibility(ctx context.Context) (EndpointCompati
 			HasRedpandaAPI:  true,
 			RedpandaFeature: "role_based_access_control",
 		},
+		{
+			URL:             consolev1alpha1connect.TransformServiceName,
+			Method:          "POST",
+			HasRedpandaAPI:  true,
```

Comment:
That alone (redpanda admin api available) doesn't indicate whether the transform service is supported though no?